public/.well-known/nostr.json

```diff
@@ -1,5 +1,6 @@
 {
   "names": {
-    "chris": "220522c2c32b3bf29006b275e224b285d64bb19f79bda906991bcb3861e18cb4"
+    "chris": "220522c2c32b3bf29006b275e224b285d64bb19f79bda906991bcb3861e18cb4",
+    "osjf": "1eef661d4e7dafc7f195ca328b0d7694fe8436e39c1e0336f80bc7e707f74d97"
```

Comment:
I'm not sure what this is for - @ChristianChiarulli can you verify that this looks right?

Reply:
this is for nip05